src/Simulation/Core/Submitters/SubmissionOptions.cs

```diff
@@ -5,32 +5,37 @@
 
 #nullable enable
 
+namespace System.Runtime.CompilerServices
+{
+    public class IsExternalInit {}
```

Comment:
As a public type, won't this conflict if another reference uses the same workaround? Can we update Microsoft.Quantum.Runtime.Core target framework to net6.0 instead?

Reply:
I've tested in other places, and that type seems to be special-cased to not conflict, allowing this as a suggested workaround (as per https://developercommunity.visualstudio.com/t/error-cs0518-predefined-type-systemruntimecompiler/1244809).